Dockerfile

```diff
@@ -20,6 +20,7 @@ WORKDIR /colcon_ws
 # hadolint ignore=SC1091
 RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    --mount=type=cache,target=/$HOME/.cache,sharing=locked \
```

Comment:
I think pip in the docker container needs this in CI, was getting errors without it.